uint8 A_LEVEL1 = 1
uint8 A_LEVEL2 = 2
uint8 A_LEVEL3 = 3
uint8 A_LEVEL4 = 4
uint8 abstraction

uint8 L_ALL = 1
uint8 L_RANGE = 1
uint8 locality
uint8 locality_min # Min range id in the plan, startin at 0
uint8 locality_max # Max range id in the plan, ending at plan length

uint8 S_GENERAL_PICTURE = 1
uint8 S_SUMMARY = 2
uint8 S_DETAILED_NARRATIVE = 3
uint8 specificity

uint8 E_LEVEL1 = 1
uint8 E_LEVEL2 = 2
uint8 E_LEVEL3 = 3
uint8 E_LEVEL4 = 4
uint8 explanation
---
bool success
